File: instructor Henry/lenguajes/Dart/act-1-06-2024.dart
import 'dart:io';

void main(){
    print("número de estudiantes:");
    int n = int.parse(stdin.readLineSync()!);

    For (i = 1; i <= n; i++) {
        print("nombre del estudiante:");
        String? s = stdin.readLineSync();

        print("ingrese las notas entre 1 y 10");

        print("nota1:");
        int n1 = int.parse(stdin.readLineSync()!);

        print("nota2:");
        int n2 = int.parse(stdin.readLineSync()!);

        print("nota3:");
        int n3 = int.parse(stdin.readLineSync()!);

        int suma = n1 + n2 + n3;
        double prom  = suma / 3;

        if (prom > 6){
            print("el estudiante $s gano");
        }else{
            print("el estudiante $s perdio");
        }
    }
}

File: instructor Henry/lenguajes/Ruby/act-1-06-2024.rb
puts "número de estudiantes:"
n = gets.chomp.to_i

for i in 1..n
  puts "nombre del estudiante:"
  s = gets.chomp

  puts "ingrese las notas entre 1 y 10"

  puts "Nota1:"
  n1 = gets.chomp.to_i

  puts "Nota2:"
  n2 = gets.chomp.to_i

  puts "Nota3:"
  n3 = gets.chomp.to_i

  suma = n1 + n2 + n3
  prom = suma / 3

  if prom > 6
    puts "El estudiante #{s} gano"
  else
    puts "El estudiante #{s} perdio"
  end
end